### ISSUE TITLE
Grid input in inline editing does not have title attribute or label

### ISSUE THREAD

**Issue by IvanDanchev** [telerik/kendo-ui-core#6520]
### Reproduction of the problem
Reproducible in:
[https://demos.telerik.com/aspnet-mvc/grid/editing-inline](https://demos.telerik.com/aspnet-mvc/grid/editing-inline)
and
[https://demos.telerik.com/aspnet-core/grid/editing-inline](https://demos.telerik.com/aspnet-core/grid/editing-inline)

1. Put a row in edit mode.
2. Inspect the input in the ProductName column

### Current behavior
The input does not have a title attribute or label.

### Expected/desired behavior
The input should either have a title attribute, like in the Kendo UI for jQuery Grid, or since we use a TextBox component as editor in MVC and Core, a label should be rendered. 

### Environment

* **Kendo UI version:** 2021.2.616
* **jQuery version:** x.y
* **Browser:** [all]

**Comment by gyankov** [comment]
Our demos follow a design that does not include a label. The editor itself is part of the component's configuration, thus everyone can configure it as per their case.

I have updated the string editors in our demos to include a title.

**Comment by MilenaCh** [comment]
Update  bool, numeric and date/time editors too
